deployment/terraform/agent.go

```diff
@@ -243,3 +232,27 @@ func (t *Terraform) initLoadtest(extAgent *ssh.ExtAgent, initData bool) error {
 
 	return nil
 }
+
+func (t *Terraform) getAppHostsFile(index int) (string, error) {
+	output, err := t.Output()
+	if err != nil {
+		return "", err
+	}
+
+	// The new entry in /etc/hosts will make SiteURL point to:
+	// - The first instance's IP if there's a single node
+	// - The proxy's IP(s) if there's more than one node
+	proxyHost := ""
+	if output.HasProxy() {
+		// This allows a bare-bones multi-proxy setup
+		// where we simply round-robin between the proxy nodes amongst the agents.
+		// Not a perfect solution, because ideally we would have used
+		// a dedicated DNS service (like Route53), but it works for now.
```

Comment:
I guess this is something we'll have to investigate once the multi-proxy setup in the app is ready for production

Reply:
What do you mean "in the app"?

Reply:
I mean the work needed to get multi-proxy setups ready for general consumption. Not sure if we'll need code changes in the server. What I mean generally is that we should use whatever our users have configured (be it Route53 or something else), but that's getting too far into the future. It was just a comment with no expected action in this PR :)

Reply:
I am supportive of the idea of unblocking testing through this simple logic, but I'd also encourage us to look into proper DNS load balancing soon enough.

Doing round robin this way wouldn't necessarily result in fair balancing either as it comes with some edge cases depending on the number of agents and proxies deployed.

Reply:
Yep, fully agree. Proper Route53 support should be something on our long term roadmap.